src/main/java/org/apache/sling/feature/cpconverter/handlers/slinginitialcontent/SlingInitialContentBundleEntryMetaDataCollector.java

```diff
@@ -106,10 +132,54 @@ Set<SlingInitialContentBundleEntryMetaData> collectFromContextAndWriteTmpFiles()
                 }
             }
         }
-
+        
         return collectedSlingInitialContentBundleEntries;
     }
+    
+    Set<SlingInitialContentBundleEntryMetaData> getDefaultContentXmlEntries() throws UnsupportedEncodingException {
```

Comment:
annotate return value with @NotNull

Reply:
`UnsupportedEncodingException` is never thrown if you remove the unnecessary clause from `createDefaultContentXMLFiles` and `addDefaultSlingContentXMLAndParents`